CraftTweaker2-API/src/main/java/crafttweaker/api/block/IBlockDefinition.java

```diff
@@ -47,16 +47,35 @@
     
     @ZenSetter("tickRandomly")
     void setTickRandomly(boolean tickRandomly);
+
+    @Deprecated
+    void setHarvestLevel(String toolClass, int level);
     
     @ZenMethod
-    void setHarvestLevel(String toolClass, int level);
+    default void setHarvestLevel(String toolClass, int level, @Optional IBlockState state) {
```

Comment:
If you want, you could always call `this.setHarvestLevel(toolClass, level)` in the default body.
So 
```java
{
    CrafttweakerAPI.logError(......);
    this.setHarvestLevel(toolClass, level);
}
```

Though your call really, as it's an edge case.